Formula/g/go-size-analyzer.rb

```diff
@@ -0,0 +1,25 @@
+class GoSizeAnalyzer < Formula
+  desc "Analyzing the dependencies in compiled Golang binaries"
+  homepage "https://github.com/Zxilly/go-size-analyzer"
+  url "https://github.com/Zxilly/go-size-analyzer.git",
+      tag:      "v1.0.4",
+      revision: "f15d7552593021ed2d3dbf7a7b40ab01fd43bcd5"
```

Comment:
```suggestion
  url "https://github.com/Zxilly/go-size-analyzer/archive/refs/tags/v1.0.4.tar.gz"
  sha256 "c92ebee6298fc1c6b25e5ea2ed0324abd047ef0ac3dc80b110030562e93f89ae"
```

Reply:
The go compiler automatically embeds vcs information at compile time, which is read by the program's version flag. If we compile with a tarball, which does not contain a .git folder, this won't work.

Reply:
yeah, we dont need that vcs info, check out other go formulae in this repo